stable/meteor/templates/cronjob.yaml

```diff
@@ -48,3 +52,10 @@ spec:
                 - key: {{ $k }}
                   path: {{ $k }}
                 {{- end }}
+            {{- range $k, $v := .Values.ssl_secrets }}
+            - name: {{ $v }}
+              secret:
+                defaultMode: 420
+                optional: true
```

Comment:
why is it optional, when we define ssl_secrets list?

Reply:
Kept it optional to avoid the total failure in case where the secret does not exist when chart is deployed for a particular recipe.